python-avd/pyavd/_eos_designs/shared_utils/wan.py

```diff
@@ -592,3 +582,20 @@ def wan_stun_dtls_profile_name(self: SharedUtils) -> str | None:
             return None
 
         return self.inputs.wan_stun_dtls_profile_name
+
+    def vrf_wan_vni(self: SharedUtils, vrf_name: str) -> int | None:
```

Comment:
only return a boolean if it is aWAN VRF as the value is used in one place only which will be reverted